### PR TITLE
fix(fe): admin 'View on website' link + donate FAQ copy (BB-37, BB-36)

### PR DESCRIPTION
**Pull request by damo-da**
## What & why

Two small front-end fixes.

### BB-37 — admin "View on website" link
The admin case editor (`AdminCaseForm`) had no way to jump to the public case page. This adds a **View on website** link in the editor header pointing at the public `/case/<slug>` route. Admin and public share the origin, so a relative link works and opens in a new tab (`target="_blank" rel="noopener"`).

The link is enabled **only when the case is `PUBLISHED`** (public). `DRAFT`/`IN_REVIEW`/`CLOSED` cases 404 publicly, so for those the control renders as a **disabled** button with a *"not public yet"* tooltip (the tooltip lives on a wrapping `<span>` because the disabled button's `pointer-events-none` forwards the hover to it). Uses the existing `ExternalLink` (lucide) icon and the shared `Button` (outline/sm) styling. New i18n keys `admin.caseForm.viewOnWebsite` / `viewOnWebsiteNotPublic` added in EN + NE.

### BB-36 — donate FAQ copy
The donate page contradicted itself: the "Ways to give" section shows a **live** Nepali bank transfer, but the FAQ answer to *"How can I support Jawafdehi financially?"* still said *"…or a Nepal bank account (coming soon)"*. Removed the `(coming soon)` in both `en.json` and `ne.json` so the copy reflects that the Nepali bank option is live.

## How verified

- **BB-36** — loaded `/donate` on the local dev server (API base → `https://api.jawafdehi.org`); the financial-support FAQ item (open by default) now reads *"…or a Nepal bank account."* with no "coming soon" (DOM assertion: `hasComingSoon=false`, `hasNepalBank=true`). Screenshot captured to `work/bug-bash-triage/evidence/round2/fixes/bb36-copy-fixed.png`.
- **BB-37** — the live admin editor is login-gated (OIDC), so it was **not** headlessly verified. Instead a new vitest component test (`src/pages/admin/jawafdehi/AdminCaseForm.test.tsx`) renders `AdminCaseForm` and asserts: a published case shows an enabled `<a href="/case/<slug>" target="_blank" rel~="noopener">`; a draft case shows **no** public link and instead a **disabled** "View on website" button carrying the "not public yet" tooltip. Also verified by code inspection.
- `bun run test` → 160 passed (30 files, incl. 2 new). `bun run lint` → clean. `bun run build:dev` → succeeded (pre-existing chunk-size warning only).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

- **New Features**
  - Added a “View on website” action to the admin case editor for published cases.
  - Added a disabled state and explanatory message when a case is not yet publicly available.

- **Content Updates**
  - Updated English and Nepali FAQ text to remove the “coming soon” note for Nepal bank account contributions.
  - Added corresponding English and Nepali labels for the new case visibility states.

### CODE REVIEW

File: src/pages/admin/jawafdehi/AdminCaseForm.tsx [hunk 1]
```diff
@@ -495,6 +495,43 @@ export default function AdminCaseForm() {
...
+                  type="button"
+                  variant="outline"
+                  size="sm"
+                  disabled
+                  aria-disabled="true"
+                >
+                  <ExternalLink className="mr-1 h-4 w-4" />
+                  {t("admin.caseForm.viewOnWebsite")}
+                </Button>
+              </span>
```
[the remark this answers is not in the document]
Comment (damo-da) [reply src/pages/admin/jawafdehi/AdminCaseForm.tsx:544]: Fixed in 3e13250. Both points addressed:

1. **Redundant ARIA** — removed `aria-disabled`; the control now relies solely on the native `<button disabled>`.
2. **Keyboard/AT accessibility** — I took a slightly different route than the `tabIndex={0}` suggestion. In practice the `title` tooltip only surfaces on *mouse hover*, not on keyboard focus, in essentially all browsers — so making the span focusable would add a tab stop that lands keyboard users on a non-interactive element that still shows nothing. Instead I kept the `title` on the wrapping span (sighted mouse users) and wired the disabled button to an `sr-only` "not public yet" hint via `aria-describedby` (generated with `useId`), which screen readers announce when reading the button. Nothing added is focusable, so it is not a focus trap, and draft cases stay disabled (no link to a public 404).

Covered by the updated `AdminCaseForm.test.tsx` (asserts no `aria-disabled`, and that `aria-describedby` resolves to the `sr-only` reason text). `bun run test` (160 pass), `bun run lint`, and `bun run build:dev` all green.

**Comment by damo-da** [comment]
### BB-37 — headless verification update (honest status)

**The live headless render DID work for the enabled (PUBLISHED) case.** Using the repo's own DEV-AUTH mechanism, I seeded an admin session into `localStorage` (`jawafdehi.devAuth.user = {is_admin:true}`, which satisfies the client-side route guard — `VITE_DEV_AUTH=true` on the dev server) and loaded the **real** `AdminCaseForm` at `/admin/jawafdehi/cases/<slug>/edit` for a live PUBLISHED case (`case-ncell-capital-gains-tax-dispute-part1`).

Observed (evidence: `work/bug-bash-triage/evidence/round2/fixes/bb37-admin-published-live.png`):
- Editor rendered (not bounced to `/admin/login`); heading **"Edit Case"**, state badge **PUBLISHED**.
- `GET /api/cases/<slug>/` → **200** (the case data loaded from the genuinely-**public** PUBLISHED read endpoint — no fake bearer involved; CORS on `api.jawafdehi.org` even returns `access-control-allow-origin: http://127.0.0.1:40114` + `allow-credentials: true`).
- The **"View on website"** link rendered with `href="/case/<slug>"`, `target="_blank"`, `rel="noopener"`, text "View on website".

**What is NOT live-verifiable (the caveat, confirmed):** the **disabled / non-public** states. DRAFT/IN_REVIEW/CLOSED case data is auth-gated at the API — a dev-auth `localStorage` snapshot mints **no real bearer**, and prod has DEV_AUTH off, so `GET /api/cases/<non-public-slug>/` returns **404** and the editor lands in its load-failure view, never rendering the form/button (evidence: `bb37-admin-nonpublic-loadfail.png`, showing `404` + no heading/state/link). So the disabled-state behavior is covered by the **component test only** — not faked as a live pass.

**Component test edge cases** (`AdminCaseForm.test.tsx`, now 5 cases, all green): PUBLISHED → enabled link; **IN_REVIEW** and **CLOSED** (added, alongside DRAFT) → disabled + `aria-describedby` "not public yet" hint + no public link; **special-character slug** → verbatim `/case/<slug>` href (consistent with every other case link in the app).

`bun run test` → **163 passed** (30 files). `bun run lint` → clean. Pushed as `3864ec0`. Not merging.

**Comment by damo-da** [comment]
⚠️ Merge-timing note: this PR was **squash-merged** (`434a625`) at commit `3e13250`, which includes the feature, the a11y fix, the donate FAQ copy, and the original 2 component-test cases — all in `main`.

My follow-up test-extension commit `3864ec0` (the **IN_REVIEW / CLOSED / special-char-slug** edge cases + the live-render verification note) was pushed to `fix/view-link-donate-copy` *after* the squash, so it is currently on the branch head but **NOT in `main`**. Flagging for a decision: if you want those extra edge-case tests in `main`, they'll need a small follow-up PR (I won't merge). Nothing behavioral is missing from `main` — it's test coverage + a comment only.
